azure_media_services/templates/player.html

```diff
@@ -27,7 +26,11 @@
         </div>
 
 % if transcript_url:
-        <div class="subtitles" data-transcript-url="${transcript_url}" style="display: none;">
+        <div class="subtitles" data-transcript-url="${transcript_url}">
+            ## This element is re-parented dynamically after the AMP is initialized and other buttons are positioned.
+            <div class="azure-media-player-toggle-button-style fa fa-quote-left toggleTranscript" role="button" aria-live="polite" tabindex="0">
+                <div class="vjs-control-content"><span class="vjs-control-text">Toggle</span></div>
+            </div>
         </div>
```

Comment:
fyi, this <div> and its children are initialized as hidden (see change to player.css)